inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -28,49 +29,59 @@
       {{ record_collection_heading() }}
     </div>
     <div id="record-title">
-      {{ record_title() }}
+      {{ render_record_title() }}
     </div>
     <div id="record-authors">
       {{ render_record_authors(is_brief=false, number_of_displayed_authors=25) }}
     </div>
+    <div id="record-year">
```

Comment:
Yes, at the moment the brief and detailed are not yet consistent.

Brief format:

![screen shot 2015-12-15 at 11 26 34 am](https://cloud.githubusercontent.com/assets/580727/11808116/cfc510d4-a31e-11e5-80b0-08b38977b8d1.png)

Detailed format:

![screen shot 2015-12-15 at 11 27 43 am](https://cloud.githubusercontent.com/assets/580727/11808131/e2f30238-a31e-11e5-870e-3bc81df9dbdd.png)

@nikpap Can you look into a smoother transition between brief and detailed page? We could consider on the detailed page having the same separate section on the right side with the same buttons (and the same labels).

For consistency we could have the same information with the date, and shortcuts to the bottom part of the page (Citations and References).